_players/mariana-podesta.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: player
+
+name: Mariana Podesta
+position: Player
+company: Robert Bosch GmbH
+bio: "LEAN Management"
+
```

Comment:
It's a good practice to always have a default avatar
`avatar: /images/players/default.png`